src/cmd-run

```diff
@@ -163,6 +163,10 @@ To exit, press Ctrl-A and then X.
 EOF
 )
 
+# generate a string like rows XX columns XX for stty
+rowcol=$(stty -a | tr ';' '\n' | grep -e 'rows\|columns' | tr '\n' ' ' )
+rowcol=$(echo "stty ${rowcol}" | base64 --wrap 0)
```

Comment:
That's...some sorcery 🧙 - I'm OK with this but probably what we should really do is default to doing SSH unless you pass `--raw-console` or something (or maybe better `--early-boot-debug`).

Rebase on master?

Reply:
Agreed, this is more of a stopgap to keep me from ripping my hair out in the meantime